documentation/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: .yarn/releases/yarn-3.2.0.cjs
```

Comment:
Is this and the `yarn/releases` folder checked in to make sure people have yarn available? If so I think would be okay to have yarn a prerequisite, if it cannot be installed as (dev-)dependency.

Reply:
I'm actually not surr anymore why I added this. I think it was because it doesn't work with older versions of yarn 🤔
I can test that again I guess 🤔

Reply:
Ok. I can't figure out why I used yarn 3.2.0. I'm going to remove it 😅

Reply:
Modern yarn puts the yarn distribution in the repo to be checked in. This has the benefit that everyone using this repo uses the exact same version.